CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 
 - Added support for multiple annotations in Code Writer API. [#1427](https://github.com/spatialos/gdk-for-unity/pull/1427)
+- Added dialog to prevent building workers with Unity Editor compile errors. [#1425](https://github.com/spatialos/gdk-for-unity/pull/1425)
```

Comment:
Its no longer a dialog, but just disabled right?

Reply:
Yes, I still need to update the Changelog